include/bluetooth/mesh/cfg_cli.h

```diff
@@ -801,6 +929,47 @@ int bt_mesh_cfg_mod_sub_get_vnd(uint16_t net_idx, uint16_t addr, uint16_t elem_a
 				uint16_t mod_id, uint16_t cid, uint8_t *status,
 				uint16_t *subs, size_t *sub_cnt);
 
+/** @brief Get the target friend's poll timeout value.
+ *
+ *  @param net_idx     Network index to encrypt with.
+ *  @param addr        Target node address.
+ *  @param lpn_address LPN address.
+ *  @param timeout     Poll timeout value.
+ *
+ *  @return 0 on success, or (negative) error code on failure.
+ */
+int bt_mesh_cfg_lpn_timeout_get(uint16_t net_idx, uint16_t addr,
+				uint16_t lpn_address, int32_t *timeout);
+
+/** @brief Get the target node's Key Refresh Phase state.
+ *
+ *  @param net_idx     Network index to encrypt with.
+ *  @param addr        Target node address.
+ *  @param key_net_idx Network key index.
+ *  @param status      Status response parameter.
+ *  @param phase       Key Refresh state.
+ *
+ *  @return 0 on success, or (negative) error code on failure.
+ */
+int bt_mesh_cfg_krp_get(uint16_t net_idx, uint16_t addr,
+			uint16_t key_net_idx, uint8_t *status, uint8_t *phase);
+
+/** @brief Set the target node's Key Refresh Phase state.
+ *
+ *  @param net_idx     Network index to encrypt with.
+ *  @param addr        Target node address.
+ *  @param key_net_idx Network key index.
+ *  @param val         New key refresh state, should be one of
+ *                     @ref BT_MESH_KR_NORMAL or BT_MESH_KR_PHASE_<*>.
+ *  @param status      Status response parameter.
+ *  @param phase       Key Refresh state.
+ *
+ *  @return 0 on success, or (negative) error code on failure.
+ */
+int bt_mesh_cfg_krp_set(uint16_t net_idx, uint16_t addr,
+			uint16_t key_net_idx, uint8_t val,
+			uint8_t *status, uint8_t *phase);
```

Comment:
Same thing here regarding not having the "current value" parameter for `set()` functions